api/v1beta1/common_types.go

```diff
@@ -5,19 +5,19 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// TemporalClusterReference is a reference to a TemporalCluster.
-type TemporalClusterReference struct {
-	// The name of the TemporalCluster to reference.
+// TemporalReference is a reference to a object.
+type TemporalReference struct {
```

Comment:
Perhaps I misunderstand what the purpose of this type is but the reason for this is that I reuse this type for namespace reference for schedules. I saw this type as more of a k8s object reference. Otherwise I could introduce a new `TemporalNamespaceReference` type instead.

Could you confirm?

Reply:
Oh ok I understand why you did this update.
Maybe we can rename it to `ObjectReference` ?

Reply:
Updated to `ObjectReference`